rewrite-java/src/main/java/org/openrewrite/java/JavaVisitor.java

```diff
@@ -926,6 +928,9 @@ public J visitWildcard(J.Wildcard wildcard, P p) {
         }
 
         setCursor(getCursor().getParent());
+        if (t == null) {
```

Comment:
if `t == null` then should `visitSpace` be invoked?

Reply:
I'd say no need to invoke visitSpace. If `t == null` that means a recipe has decided to remove t. Preserving the whitespace of a removed element would be a more surprising result than removing an element's spacing along with it.